src/pybind/rbd/rbd.pyx

```diff
@@ -898,13 +898,12 @@ class RBD(object):
     def trash_move(self, ioctx, name, delay=0):
         """
         Move an RBD image to the trash.
-
```

Comment:
correct, but I get an error when I leave that newline in:

> docstring of rbd.RBD.trash_move:8:Field list ends without a blank line; unexpected unindent.

Perhaps there is a better fix that leaves the newline in?